Check/DoctrineDbal.php

```diff
@@ -21,7 +21,12 @@ public function check()
     {
         $connection = $this->manager->getConnection($this->connectionName);
         $query = $connection->getDriver()->getDatabasePlatform()->getDummySelectSQL();
-        $connection->fetchColumn($query);
+
+        if (method_exists($connection, 'fetchOne')) {
```

Comment:
Can you add a comment here to explain why this is needed?

Reply:
Done